composer.json

```diff
@@ -8,19 +8,23 @@
         {
             "name": "Toon Verwerft",
             "email": "toon.verwerft@phpro.be"
+        },
+        {
+            "name": "Jelle Deneweth",
+            "email": "jelle.deneweth@gmail.com"
         }
     ],
     "require": {
         "php": "^8.0",
         "ext-json": "*",
         "amphp/amp": "^2.5",
         "azjezz/psl": "~1.9.1",
-        "league/uri": "^6.3",
+        "league/uri": ">= 6.3 <=6.7",
         "php-http/client-common": "^2.3",
         "php-http/discovery": "^1.12",
         "php-http/httplug": "^2.2",
         "php-http/logger-plugin": "^1.2",
-        "php-http/message": "^1.9",
+        "php-http/message": ">=1.9 <=1.12",
```

Comment:
I would rather not set a maximum supported version but instead go with something that works on all newer versions as well.
Can we find a solution for the HttpFormatter thing that works on all versions?

(In any case: if we set a higher bound, composer might just install a lower version of http-tools and combine it with http/message 1.13. - That would be correct since we state that we support everything >=1.9 in previous version)

Reply:
Will have to check if the method exists in the newly introduced formatResponseForRequest.
Also will update tests and remove constraint.